controller-server/src/main/java/com/yahoo/vespa/hosted/controller/restapi/gui/GUIHandler.java

```diff
@@ -0,0 +1,116 @@
+// Copyright 2018 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.vespa.hosted.controller.restapi.gui;
+
+import com.google.inject.Inject;
+import com.yahoo.container.jdisc.HttpRequest;
+import com.yahoo.container.jdisc.HttpResponse;
+import com.yahoo.container.jdisc.LoggingRequestHandler;
+import com.yahoo.vespa.hosted.controller.restapi.ErrorResponse;
+import com.yahoo.yolean.Exceptions;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.file.Files;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.logging.Level;
+
+
+public class GUIHandler extends LoggingRequestHandler {
+
+    @Inject
+    public GUIHandler(Context parentCtx) {
+        super(parentCtx);
```

Comment:
What's "Context"? A class you forgot to add?

Reply:
Ah, I see. You don't have to, but it's ok to leave it like that ... didn't know about that class.